README.md

```diff
@@ -520,31 +520,6 @@ Unless specified by using the parameter `pricingSource`, the [GET /tokens/prices
 
 Internally, the Coingecko pricing source looks for the price quote in the its pricing cache and return it if it exists. Otherwise, it fetches the price from the Coingecko API endpoint and store it in the cache with an expiration time specified in the config.json file.
 
-### Algorithm
```

Comment:
Note to reviewer: this has been moved to `ingest.md` with light changes

Reply:
🆗